apps/api/src/app/workflows-v1/notification-template.controller.ts

```diff
@@ -108,7 +114,10 @@ export class NotificationTemplateController {
         description: body.description,
         workflowId: body.identifier,
         critical: body.critical,
-        preferenceSettings: body.preferenceSettings,
+        defaultPreferences: DEFAULT_WORKFLOW_PREFERENCES,
+        userPreferences:
+          body.preferenceSettings &&
+          buildWorkflowPreferencesFromPreferenceChannels(body.critical, body.preferenceSettings),
```

Comment:
Should this code be identical to [this line](https://github.com/novuhq/novu/pull/7005/files#diff-c995a3ecb137586ca535a06504687da459250bbc1a11ecf9a28c11bb0b25bc5dR205)?

Reply:
Yes it should, good catch.

Reply:
Updated.